frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -503,71 +504,50 @@ export const supportLogic = kea<supportLogicType>([
-                const hasBoostTrial = billing.trial?.status === 'active' && (billing.trial?.target as any) === 'boost'
-                const hasScaleTrial = billing.trial?.status === 'active' && (billing.trial?.target as any) === 'scale'
-                const hasEnterpriseTrial = billing.trial?.status === 'active' && billing.trial?.target === 'enterprise'
-
-                // Update the checks to include trials
-                const hasBoostAddonOrTrial = hasBoostAddon || hasBoostTrial
-                const hasScaleAddonOrTrial = hasScaleAddon || hasScaleTrial
-                const hasEnterpriseAddonOrTrial = hasEnterprisePlan || hasEnterpriseTrial
-
-                // Determine plan level based on priority
-                if (hasEnterpriseAddonOrTrial) {
-                    planLevelTag = 'plan_enterprise'
-                } else if (isNewOrganization) {
-                    planLevelTag = 'plan_onboarding'
-                } else if (hasScaleAddonOrTrial) {
-                    planLevelTag = 'plan_scale'
-                } else if (hasTeamsAddon) {
-                    planLevelTag = 'plan_teams_legacy'
-                } else if (hasBoostAddonOrTrial) {
-                    planLevelTag = 'plan_boost'
-                } else if (currentPlan?.plan_key?.includes('paid')) {
-                    planLevelTag = 'plan_pay-as-you-go'
-                } else if (currentPlan?.plan_key?.includes('free') || billing.subscription_level === 'free') {
-                    planLevelTag = 'plan_free'
+            let planLevelTag = 'plan_free'
+
+            const knownEnterpriseOrgIds = ['018713f3-8d56-0000-32fa-75ce97e6662f']
+            const isKnownEnterpriseOrg = knownEnterpriseOrgIds.includes(userLogic?.values?.user?.organization?.id || '')
+
+            const orgCreatedAt = currentOrganization?.created_at
+            const isNewOrganization = orgCreatedAt && dayjs().diff(dayjs(orgCreatedAt), 'month') < 3
+
+            const hasBoostTrial = billing?.trial?.status === 'active' && (billing.trial?.target as any) === 'boost'
+            const hasScaleTrial = billing?.trial?.status === 'active' && (billing.trial?.target as any) === 'scale'
+            const hasEnterpriseTrial = billing?.trial?.status === 'active' && billing.trial?.target === 'enterprise'
+
+            if (isKnownEnterpriseOrg || hasEnterpriseTrial || billingPlan === BillingPlan.Enterprise) {
+                planLevelTag = 'plan_enterprise'
+            } else if (isNewOrganization) {
+                planLevelTag = 'plan_onboarding'
+            } else if (hasScaleTrial) {
+                planLevelTag = 'plan_scale'
+            } else if (hasBoostTrial) {
+                planLevelTag = 'plan_boost'
+            } else if (billingPlan) {
+                switch (billingPlan) {
+                    case BillingPlan.Enterprise:
+                        planLevelTag = 'plan_enterprise'
+                        break
```

Comment:
This can be removed - it's causing TS error, since we're already checking for this case earlier on line 519